File: leet-code/findNumbers.go
package main

func findNumbers(nums []int) int {
	var count int
	for _, val := range nums {
		c := 0
		for val > 0 {
			val /= 10
			c++
		}
		if c%2 == 0 {
			count++
		}
	}
	return count
}

File: leet-code/findNumbers.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function (nums) {
  let count = 0
  for (const val of nums) {
    if (String(val).length % 2 === 0) {
      count++
    }
  }
  return count
}

var findNumbers = function (nums) {
  let count = 0
  for (let val of nums) {
    let c = 0
    while (val > 0) {
      val = parseInt(val / 10)
      c++
    }
    if (c % 2 == 0) {
      count++
    }
  }
  return count
}
